client/src/partials/bhima/application.js

```diff
@@ -11,12 +11,19 @@ ApplicationController.$inject = [
  * This top-level controller is currently  responsible for initializing language
  * loading and controlling the side-bar hide/show methods.
  */
-function ApplicationController(AppCache, Session, Languages, $state, $rootScope) {
+function ApplicationController(AppCache, Session, Languages, $state, $rootScope, Notify) {
   var vm = this;
 
   // load in the application cache
   var cache = AppCache('preferences');
 
+  // Default sidebar state
+  /** @todo Load sidebar state before angular is bootstraped to remove 'flicker' */
+  vm.sidebarExpanded = false;
```

Comment:
This is a duplication of line 37 in this same file - potentially broken during a merge or rebase.  Could you remove it and the comments?